### PR TITLE
Added guide for Pyramid

### PR DESCRIPTION
**Pull request by sourabh1031**
NOTE:
- Please fill out the template below 

## Did you read the instructions in the [README](https://github.com/the-ethan-hunt/first-timers-guide/blob/master/README.md)?

> Y

## Have you read the instructions in [CONTRIBUTING](https://github.com/the-ethan-hunt/first-timers-guide/blob/master/CONTRIBUTING.md)?

> Y

## Have you written this article by yourself? Have you taken inspiration from anywhere and then written it in your words?

> [I have taken the reference from [documentation](https://docs.pylonsproject.org/projects/pyramid/en/latest/narr/introduction.html#)]

### CODE REVIEW

File: guides/Web_Frameworks/Pyramid.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Pyramid
+Pyramid is python web framework used to create web applcations. It is mostly used in projects that require flexibility, scalability and speed compared to django,flask or others.It is a product of pylons project.Another coolest thing in pyramid is that it is open source.It is also called down to earth framework because of its structure and it is clearly visible from its logo as well.
```
Comment (RN0311) [inline guides/Web_Frameworks/Pyramid.md]: 1) Spell check of "applications" in very first line.

File: guides/Web_Frameworks/Pyramid.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+After installing python3 there is no much tedious task to install pyramid
+First create a virtual environment into your project directory by using the following command. It is useful so that you don't need to install the packages system wide. To download pyramid for all platforms just head over to this [link](https://docs.pylonsproject.org/projects/pyramid/en/latest/narr/install.html). It provides detailed instructions related to installation on all the platforms.
+The github repo of pyramid can be visited from [here](https://github.com/Pylons/pyramid).If you want to contribute to the project then you are welcome to do so as it is very great and welcoming community.
+
+## Learning Resources
+The Pylon community has provided has very rich [documentation](https://docs.pylonsproject.org/projects/pyramid/en/latest/narr/introduction.html#).If you want to get your hands dirty with the framework it is the first and best step to proceed.
```
Comment (RN0311) [inline guides/Web_Frameworks/Pyramid.md]: 2) Remove double "has" from the first line.

**Comment by sourabh1031** [comment]
@RN0311 i have updated the guide please review it

**Comment by RN0311** [comment]
@the-ethan-hunt : Reviewed.

**Pull request merged**